numpydoc/tests/test_docscrape.py

```diff
@@ -720,17 +720,6 @@ def test_parameters_without_extended_description():
     """)
 
 
-def test_escape_stars():
-    signature = str(doc3).split('\n')[0]
-    assert signature == r'my_signature(\*params, \*\*kwds)'
-
-    def my_func(a, b, **kwargs):
-        pass
-
-    fdoc = FunctionDoc(func=my_func)
-    assert fdoc['Signature'] == r'my_func(a, b, \*\*kwargs)'
```

Comment:
This might actually be important. Do things with `*args, **kwargs` render properly now? That's what this is meant to check/ensure. Can we replace this test rather than just remove it? Maybe check that the RST generated has the `*`s escaped properly?